x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/types.ts

```diff
@@ -13,6 +13,19 @@ export const RuleTypeSchema = t.keyof({
 });
 export type RuleType = t.TypeOf<typeof RuleTypeSchema>;
 
+/**
+ * Params is an "record", since it is a type of AlertActionParams which is action templates.
+ * @see x-pack/plugins/alerting/common/alert.ts
+ */
+export const action = t.exact(
+  t.type({
+    group: t.string,
+    id: t.string,
+    action_type_id: t.string,
+    params: t.record(t.string, t.any),
```

Comment:
`t.any` [is deprecated](https://github.com/gcanti/io-ts/blob/master/CHANGELOG.md#171) it looks like and `t.unknown` is recommended instead. However I think we may be able to just use `UnknownRecord` ala `params: t.UnknownRecord,` -- thoughts?